src/plugins/workspace/public/components/workspace_menu/workspace_menu.tsx

```diff
@@ -185,7 +180,138 @@ export const WorkspaceMenu = ({ coreStart }: Props) => {
       panelPaddingSize="none"
       anchorPosition="downCenter"
     >
-      <EuiContextMenu initialPanelId={0} panels={panels} />
+      <EuiPanel paddingSize="m" hasBorder={false} color="transparent">
+        <EuiFlexGroup justifyContent="center" alignItems="center" direction="column" gutterSize="s">
+          {currentWorkspace ? (
+            <>
+              <EuiFlexItem grow={false}>
+                <EuiAvatar
+                  size="m"
+                  type="space"
+                  name={currentWorkspaceName}
+                  color={currentWorkspace?.color}
+                  initialsLength={2}
+                />
+              </EuiFlexItem>
+              <EuiFlexItem grow={false} data-test-subj="context-menu-current-workspace-name">
+                {currentWorkspaceName}
+              </EuiFlexItem>
+              <EuiFlexItem grow={false} data-test-subj="context-menu-current-use-case">
+                {getUseCase(currentWorkspace)}
```

Comment:
The result of `getUseCase` will be a use case id right? Shall we map to the real use case name here?

Reply:
Sure I will update the code.